### DISCUSSION TITLE
Dreiphasig messen - welche (Aktoren) können das?

### DISCUSSION THREAD

**Discussion by ElectricPerfum** [camueller/SmartApplianceEnabler#313] [Installation und Betrieb]
Da ich meine Drehstromwasserpumpe gerne im SunnyPortal darstellen würde frage ich mich mit welchem Messaktor ich das bewerkstelligen könnte. Natürlich wäre mir eine Abfrage via http oder Modbus am liebsten.
Habe in der DOC keinen Hinweis auf dreiphasige Verbraucher im Zusammenhang mit deren Erfassung gefunden.
Vielen Dank für zielführende Hinweise.

**Comment by premultiply** [comment]
Jeder stinknormale Modbus-Zähler wie z. B. der SDM72DM-V2 ?

**Comment by ElectricPerfum** [comment]
vielen Dank für die schnelle Antwort.
leider habe ich die Frage ganz offensichtlich nicht präzise genug gestellt denn ich hätte den Zähler selbst natürlich am liebsten gleich mit integrierter LAN Schnittstelle. Gibt es das auch oder brauch ich da wieder einen extra Wandler dazu ?

**Reply by premultiply** [reply]
Steht doch oben dass du gerne Modbus hättest...

USB-Schnittstellenwandler für 3 EUR, zwei 120 Ohm Widerstände, verdrillte Doppelader, fertig.

Kannst es natürlich auch auf TCP/IP umsetzen mit einem dieser zahlrichen RS485-TCP-Wandler.
Da musst du aber dann schon 20 bis 30 EUR für kalkulieren.
Wirklich brauchen tut man das aber in den seltensten Fällen bei abgesetzten Installationen.

RS485 geht auch über eine Doppelader einer LAN-Verkabelung falls du da Berührungsängste hast...

**Reply by ElectricPerfum** [reply]
ich danke Dir nochmals und ja Modbus oder http habe ich geschrieben.
gibt es denn ein Gerät welches die abgenommene Leistung aufzeichnen kann und auf simple weise in den SAE gebracht werden kann?
ich finde dazu leider nichts in der Doku außer unterstützte Geräte. Die Shellys gibt´s ja leider nur 1 phasig, daher dachte ich an sowas wie mit Hutschiene aber hizu kenn ich mich zu wenig aus und daher bin ich sehr dankbar für die infos dazu.
am liebsten wäre mir hutschienenadapter mit LAN-anschluss wie ein SHM halt.

**Reply by camueller** [reply]
Der Zähler muss nichts aufzeichnen, er muss nur durch den SAE abgefragt werden können. Dreiphasige Zähler für die Hutschiene gibt es zur Genüge. Zähler mit Modbus sind genauer, aber etwas teurer, als solche mit S0-Ausgang (Impulse), welche nur auf einem Raspberry Pi funktionieren.

**Reply by ElectricPerfum** [reply]
okay verstehe, aber S0 Signal wird dann wie abgegriffen vom Raspberry PI wenn nicht über LAN ?
brauchts da wieder ein AddOn Modul dafür ?
Ich kenne die Hardware hier nicht.

**Reply by camueller** [reply]
Offensichtlich ist Dir Deine Zeit zu schade, mal einen Blick in die Dokumentation zu werfen. Meine Zeit ist mir zu schade, solche Fragen zu beantworten.

**Reply by ElectricPerfum** [reply]
wieso sollte mir die zeit dazu zu schade sein? dann würde ich doch nicht hier danach fragen und noch dazu darauf hinweisen daß ich den zusammenhang nicht erkennen kann.
natürlich habe ich die doku sehr wohl durchforstet, bin aber auf keinen grünen zweig gekommen.
zumal ich aber nicht weis welches gerät ich hier überhaupt zum einsatz bringen kann, was ja nun so halbwegs geklärt ist, also S0 oder eben Modbus wird schreitet das Projekt Erfassung der Daten fort. Zumindest ein Hinweis in welchem Abschnitt ich suchen soll wäre doch hilfreich.
dir auch freundliche Grüße 

PS: vielen dank an premultiply für den Hinweis: https://github.com/camueller/SmartApplianceEnabler/blob/master/doc/ModbusMeter_DE.md

**Comment by DrMabuse100** [comment]
Hallo gibts es einen Zähler der wo Modbus und S0 gleichzeitig hat und auch gleichzeitig ausgebeb könnte ?

**Comment by ElectricPerfum** [comment]
Der SDM72DM-V2 hat offensichtlich beides aber ob Du beides gleichzeitig abgreifen kannst ist mir nicht bekannt.
S0 gibt meines wissens Impulse ab, über Modbus kann direkt abgefragt werden.
